drivers/hmis/spec/models/hmis/hmis_data_cleanup_util_spec.rb

```diff
@@ -9,6 +9,15 @@
 
 RSpec.describe HmisDataCleanup::Util, type: :model do
   let!(:hmis_ds) { create :hmis_data_source }
+  let(:today) { Date.current }
+  let(:yesterday) { today - 1.day }
+  let(:default_enrollment_attrs) do
```

Comment:
this could dry up the rest of the factory invocations but I didn't want to break any tests ahead of launch